modules/blazium_sdk/doc_classes/BlaziumClient.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="BlaziumClient" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
 	<brief_description>
-		An abstract base node used to connect to a blazium services.
+		An abstract base node used to connect to blazium services.
```

Comment:
Blazium should probably be capitalized since there are already changes being made here.